com.unity.render-pipelines.universal/Editor/Camera/CameraStackReorderableList.cs

```diff
@@ -0,0 +1,301 @@
+        {
+            var cam = serializedCamera[list.index];
+            if (cam == null)
+                return;
+
+            if (Event.current.clickCount == 2)
+            {
+                Selection.activeObject = cam;
+            }
+
+            EditorGUIUtility.PingObject(cam);
+        }
+
+        void DrawElementCallback(Rect rect, int index, bool isActive, bool isFocused)
+        {
+            rect.height = EditorGUIUtility.singleLineHeight;
+            rect.y += 1;
+
+            var element = serializedCamera.cameras.GetArrayElementAtIndex(index);
+
+            var cam = element.objectReferenceValue as Camera;
+            if (cam != null)
+            {
+                bool outputWarning = !Validate(cam, () => !IsStackCameraOutputDirty(cam), outputWarningCameras);
+
+                GUIContent nameContent =
+                    outputWarning ?
+                    EditorGUIUtility.TrTextContent(cam.name, "Output properties do not match base camera", CoreEditorStyles.warningIcon) :
+                    EditorGUIUtility.TrTextContent(cam.name);
+
+                var type = cam.gameObject.GetComponent<UniversalAdditionalCameraData>().renderType;
+                bool typeError = !Validate(cam, () => type == CameraRenderType.Overlay, typeErrorCameras);
+                GUIContent typeContent =
+                    typeError ?
+                    EditorGUIUtility.TrTextContent(type.GetName(), "Not a supported type", CoreEditorStyles.errorIcon) :
+                    EditorGUIUtility.TrTextContent(type.GetName());
+
+                EditorGUI.BeginProperty(rect, GUIContent.none, element);
+                var labelWidth = EditorGUIUtility.labelWidth;
+                EditorGUIUtility.labelWidth -= 20f;
+
+                using (var iconSizeScope = new EditorGUIUtility.IconSizeScope(new Vector2(rect.height, rect.height)))
+                {
+                    EditorGUI.LabelField(rect, nameContent, typeContent);
+                }
+
+                // Printing if Post Processing is on or not.
+                var isPostActive = cam.gameObject.GetComponent<UniversalAdditionalCameraData>().renderPostProcessing;
+                if (isPostActive)
+                {
+                    Rect selectRect = new Rect(rect.width - 20, rect.y, 50, EditorGUIUtility.singleLineHeight);
+
+                    EditorGUI.LabelField(selectRect, "PP");
+                }
+                EditorGUI.EndProperty();
+
+                EditorGUIUtility.labelWidth = labelWidth;
+            }
+            else
+            {
+                camera.GetComponent<UniversalAdditionalCameraData>().UpdateCameraStack();
+
+                // Need to clean out the errorCamera list here.
+                ClearValidation();
+            }
+        }
+
+        // Modified version of StageHandle.FindComponentsOfType<T>()
+        // This version more closely represents unity object referencing restrictions.
+        // I added these restrictions:
```

Comment:
any reason why we have those restrictions? i would have imagined that if you have multiple scenes added additively, if each one has a camera then you can still do the stacking

Reply:
I do not know, maybe @lukaschod can give us some light on this

Reply:
@lukaschod any idea?